### PR TITLE
Task 55 profile front end

### PR DESCRIPTION
**Pull request by universezzz**

### CODE REVIEW

File: src/containers/profile/profileTab/index.jsx [hunk 1]
```diff
@@ -1,96 +1,212 @@
...
+  useEffect(() => {
+    getProfileRequest();
+  }, []);
+
+  useEffect(() => {
+    if (profile) {
+      setFormData(profile);
+    }
+  }, [profile])
+
```
Comment (slavasgod) [inline src/containers/profile/profileTab/index.jsx:63]: 2 useEffects?

File: src/containers/profile/profileTab/index.jsx [hunk 2]
```diff
@@ -1,96 +1,212 @@
...
+  const onChange = e => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+
```
Comment (slavasgod) [inline src/containers/profile/profileTab/index.jsx:67]: Why you destructure `formData` here? On every word tapping you make a destructure of the object, I think it's bad

File: src/services/auth.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { httpService } from './httpService';
 
 const config = {
   headers: { 'Content-Type': 'application/json' },
+  Authorization: localStorage.token,
```
Comment (slavasgod) [inline src/services/auth.js]: Remove authorization from config

**Review by slavasgod — CHANGES_REQUESTED** [review]

File: src/containers/profile/profileTab/index.jsx (hunk 1 above)
Comment (universezzz) [reply src/containers/profile/profileTab/index.jsx:63]: <3 everything's okay

**Review by slavasgod — APPROVED** [review]

File: src/containers/profile/profileTab/styles.module.scss [hunk 4]
```diff
@@ -5,7 +5,7 @@
 @import '../../../scss/mixins/input.scss';
 
 .profile {
-  margin: 0 15;
+  margin: 0 15px;
   margin-top: 40px;
```
Comment (OlegPanasyuk) [inline src/containers/profile/profileTab/styles.module.scss]: Maybe ,` margin: 40px 15px 0 15px;`

File: src/containers/profile/profileTab/styles.module.scss [hunk 5]
```diff
@@ -21,43 +21,55 @@
...
+    margin-top: 15px;
 
     input {
       @include primary-input;
       width: 100%;
-      padding: 15px 5px;
+      padding: 6px 5px;
+      margin: 15px 0;
     }
     margin-right: 12px;
```
Comment (OlegPanasyuk) [inline src/containers/profile/profileTab/styles.module.scss:54]: As I believe that we need to group familiar blocks like margin:

```
margin-top: 15px;
margin-right: 12px;
```

**Review by OlegPanasyuk — APPROVED** [review]

**Pull request merged**